src/rgw/rgw_op.cc

```diff
@@ -6135,9 +6136,15 @@ void RGWInitMultipart::execute(optional_yield y)
   if (rgw::sal::Object::empty(s->object.get()))
     return;
 
+  if (multipart_trace) {
```

Comment:
when would "multipart_trace" be null?

Reply:
just in case an error occurred in opentelemetry `StartSpan`. I thought it may be safer to check it. so in case of failure in tracing we won't affect the multipart upload op.